bitnami/airflow/templates/_helpers.tpl

```diff
@@ -220,6 +220,25 @@ Add environment variables to configure database values
 {{- ternary .Values.postgresql.postgresqlDatabase .Values.externalDatabase.database .Values.postgresql.enabled | quote -}}
 {{- end -}}
 
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "airflow.database.existingsecret.key" -}}
+{{- if (.Values.postgresql.enabled) -}}
```

Comment:
You don't need the brackets. Apply to the rest of the `if` statements

Reply:
Done, thank you!